### ISSUE TITLE
Update Remote's GUI to match NVDA

### ISSUE THREAD

**Issue by SaschaCowley** [nvaccess/nvda#17785]
### Is your feature request related to a problem? Please describe.

The UI and UX of Remote (#17580) are somewhat incongruent with the rest of NVD

* The settings and connection dialogs use radio buttons, which are not commonly used in NVDA. These radio button groups are also not labelled.
* The settings and connection dialogs are not documented in the user guide
* The connection dialog appears at the top left of the screen, unlike the rest of our dialogs which appear centered on the screen
* The Remote menu has separate options for connect and disconnect, whereas we typically present only the available action (ex. in the add-on store, "install" and "uninstall" actions are not shown at once)
* The connect and disconnect gestures take up 2 keyboard shortcuts, despite being mutually exclusive.

### Describe the solution you'd like

Update the UX of Remote to match NVDA's UX:
